README.md

```diff
@@ -25,44 +25,44 @@ WasmEdge (previously known as SSVM) is a lightweight, high-performance, and exte
 
 ## Features
 
-WasmEdge can run standard WebAssembly bytecode programs compiled from C/C++, Rust, Swift, AssemblyScript, or Kotlin source code. It also [runs JavaScript](https://wasmedge.org/book/en/dev/js.html) through an embedded [QuickJS engine](https://github.com/second-state/wasmedge-quickjs). WasmEdge supports all standard WebAssembly features and proposed extensions. It also supports a number of extensions tailored for cloud-native and edge computing uses (e.g., the [WasmEdge Tensorflow extension](https://www.secondstate.io/articles/wasi-tensorflow/)). 
+WasmEdge can run standard WebAssembly bytecode programs compiled from C/C++, Rust, Swift, AssemblyScript, or Kotlin source code. It [runs JavaScript](https://wasmedge.org/book/en/dev/js.html), including 3rd party ES6, CJS, and NPM modules, in a secure, fast, lightweight, portable, and containerized sandbox. It also supports mixing of those languages (e.g., to [use Rust to implement a JavaScript API](https://wasmedge.org/book/en/dev/js/rust.html)).
 
-* [WebAssembly standard extensions](docs/extensions.md#webassembly-standard-extensions)
-* [WasmEdge extensions](docs/extensions.md#wasmedge-extensions)
-
-WasmEdge extensions to WebAssembly are typically available to developers as Rust SDKs or [JavaScript APIs](https://wasmedge.org/book/en/dev/js/quickstart.html). 
+WasmEdge supports all standard WebAssembly features and many proposed extensions. It also supports a number of extensions tailored for cloud-native and edge computing uses (e.g., the [WasmEdge network sockets](https://wasmedge.org/book/en/dev/rust/networking.html), and the [WasmEdge Tensorflow extension](https://wasmedge.org/book/en/dev/rust/tensorflow.html)).
 
 ## Integrations
 
 WasmEdge and its contained wasm program can be started from the [CLI](https://wasmedge.org/book/en/index.html) as a new process, or from a existing process. If started from an existing process (e.g., from a running [Node.js](https://wasmedge.org/book/en/embed/node.html) or [Go](https://wasmedge.org/book/en/embed/go.html) or [Rust](bindings/rust/wasmedge-rs) program), WasmEdge will simply run inside the process as a function. Currently, WasmEdge is not yet thread-safe. In order to use WasmEdge in your own application or cloud-native frameworks, please refer to the guides below.
 
 * [Embed WasmEdge into a host application](https://wasmedge.org/book/en/embed.html)
 * [Orchestrate and manage WasmEdge instances using container tools](https://wasmedge.org/book/en/kubernetes.html)
+* [Run a WasmEdge app as a Dapr microservice](https://wasmedge.org/book/en/frameworks/mesh/dapr.html)
+* [Use Reactr to embed and extend WasmEdge functions in SaaS](https://wasmedge.org/book/en/frameworks/app/reactr.html)
 
 # Community
 
 ## Contributing
 
-If you would like to contribute to the WasmEdge project, please refer to our [CONTRIBUTING](docs/CONTRIBUTING.md) document for details. If you are looking for ideas, checkout our [wish list](docs/wish_list.md)!
```

Comment:
Looks like we will no longer use this file `docs/wish_list.md`. Should we remove it directly?

Reply:
I think we still leave that file in place because it might be linked to from articles we published on external websites.

Reply:
How about adding a deprecated message on `wish_list.md` with a new link to our book?

Reply:
I updated the wish list